src/Attributes/Wheres.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Spatie\RouteAttributes\Attributes;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_CLASS)]
+class Wheres implements RouteAttribute
```

Comment:
Rename this to `Where`, feels much more natural to me.

Reply:
I named it Where first but due to it takes multiple arguments i renamed it to wheres.
There is also the possibility to implement it with multiple attributes.

e.g.:

```
    #[Get('my-where-method/{param}/{param2}')]
    #[Where('param2', '[a-zA-Z]+'], Where('param', '[0-9]+')]
    public function myWhereMethod()
    {

    }
```

This leads to more complicated resolving strategy (collecting multiple attributes) but feels natural when using.

Reply:
Let's optimise for that natural feeling when using it.

Reply:
I implemented the single Where version and some convenience attributes. I updated the docs.